syncr_backend/node_init.py

```
@@ -0,0 +1,59 @@
+import os
+import shutil
+
+from syncr_backend import crypto_util
+
+
+def force_initialize_node() -> None:
+    """Initialize new node in .node directory
+    and overwrite existing .node dir"""
+
+    if os.path.exists(".node"):
+        shutil.rmtree(".node")
+
+    initialize_node()
+
+
+def is_node_initialized() -> bool:
+    return os.path.exists(".node")
+
+
+def initialize_node() -> None:
+    """Initialize new node in .node directory
+    Create the private key file"""
+    try:
+        if os.path.exists(".node"):
+            raise FileExistsError
+
+        os.mkdir(".node")
+        private_key = crypto_util.generate_private_key()
+        write_private_key_to_disk(private_key)
+
+    except (FileExistsError):
+        print("Error: node already initiated")
+
+
+def write_private_key_to_disk(key: crypto_util.rsa.RSAPrivateKey) -> None:
+    """Write Private Key (and public key attached) to file"""
+    try:
+        if os.path.exists(".node/private_key.pem"):
+            raise FileExistsError
+
+        with open(".node/private_key.pem", "wb") as keyfile:
+            keyfile.write(crypto_util.dump_private_key(key))
+            keyfile.close()
+    except (FileNotFoundError):
```

you should just raise these errors